XamarinCommunityToolkit/Converters/StringIsNotNullOrEmptyConverter.shared.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    public class StringIsNotNullOrEmptyConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) => !string.IsNullOrEmpty(value?.ToString());
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
How about to create a base internal abstract class "BaseConverter" with non-implemented "ConvertBack". It will allow us to not copy-paste "ConvertBack" everywhere

Reply:
I wouldn't increase the object dependency tree (which brings a minimal performance hit) just to prevent a NotImplementedException, which you don't even have to copy as it's generated by default by implementing the interface.

Reply:
> I wouldn't increase the object dependency tree (which brings a minimal performance hit) just to prevent a NotImplementedException, which you don't even have to copy as it's generated by default by implementing the interface.

I believe this performance hit is insignificant.

Reply:
I tend to agree with @bartlannoeye here.

Reply:
I also agree that it brings more clarity to have it say right there where it matters that that conversion back is not possible instead of hiding it away in a base class for the sake of abstraction.